apps/api/lib/api/user/user.ex

```diff
@@ -3,11 +3,29 @@ defmodule API.User do
   Context for users
   """
 
+  @case_insensitive_params ["username", "email"]
+
   alias API.User.{Authorization}
 
   def create(attrs) do
-    DB.User.create(attrs)
+    attrs
+    |> case_insensitive
+    |> DB.User.create
+  end
+
+  def login(attrs) do
+    attrs
+    |> case_insensitive
+    |> Authorization.authorize
   end
 
-  def login(attrs), do: Authorization.authorize(attrs)
+  defp case_insensitive(attrs) do
```

Comment:
does this mean we're essentially ignoring the case that the user gives for their username? It's definitely more work to try and maintain that but people do like their capital letters in usernames